jade/index.jade

```diff
@@ -0,0 +1,61 @@
+doctype html
+html(lang='en')
+  head
+    meta(charset='utf-8')
+    meta(name='description', content='meta desc')
+    meta(name='viewport', content='width=device-width')
+    title Avsc
+    link(rel='stylesheet', href='../stylesheets/main.css', type='text/css')
+  
+  body
+    script(src='../js/bundle.js')
+    div.container
+      table
+        col.wide
+        col.narrow
+        col.wide
+        tr.input-row
+          td
+            div.outer
+              div
+                textarea#schema.textbox.verifiable
```

Comment:
Browsers don't complain but it seems having `textarea` elements outside a `form` (or without a `form` attribute) is [illegal](http://stackoverflow.com/q/14528951/1062617)!